rust-toolchain.toml

```diff
@@ -1,2 +1,4 @@
 [toolchain]
-channel = "nightly"
+channel = "nightly-2024-09-23"
+components = ["rust-src"]
+targets = ["wasm32-unknown-unknown"]
```

Comment:
Are these required ? @rupansh-gob

Reply:
@adarsh-rs can we remove these ? Don't think are required

Reply:
@komal-sai-yral yes , i tested without adding this, it works fine